Makefile

```diff
@@ -1,11 +1,195 @@
-all:
-	@echo "Please change into the directory docs/ before running any make commands"
+# Makefile for Sphinx documentation
+#
 
-help:
-	@echo "Please change into the directory docs/ before running any make commands"
+# You can set these variables from the command line.
+SPHINXOPTS    = 
+SPHINXBUILD   = sphinx-build
+PAPER         =
+DOCS_DIR      = ./docs/
+BUILDDIR      = ../_build/
```

Comment:
Can you verify the `BUILDDIR` is correct, now that the Makefile has moved up a level? I think when the docs are deployed, its location matters.

Reply:
@ericof might know, or it might be in the GitHub Actions workflow.

Reply:
Stupid me moved and modified in one commit. So it is not obviously that I changed the BUILDDIR. The build directory _build is, same as before, in /.

Reply:
Here it is:

https://github.com/plone/training/blob/695659fb79d92373664fe805d2f08e9ffbf7bde5/.github/workflows/build_deploy.yml#L44

Reply:
If the builddir is still in the same location as before, then we're good.

Reply:
No, i did not change. But as the dir is changed, than build is run, the _build directory is in place.

Reply:
Got it. Please merge.